libs/sdk-core/src/models.rs

```diff
@@ -1098,6 +1109,23 @@ pub struct Channel {
     ///
     /// This may be empty for older closed channels, if it was not possible to retrieve the closing txid.
     pub closing_txid: Option<String>,
+
+    pub htlc: Vec<Htlc>,
```

Comment:
I think this should be plural
```suggestion
    pub htlcs: Vec<Htlc>,
```